etcdctl/ctlv2/ctl.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/urfave/cli"
 )
 
-func Start() {
+func StartWithError() error {
```

Comment:
Just a suggestion: An alternative would be to use the "Start" (returns error) and "MustStart" (does not return an error). This is a idiom used commonly in go.

Reply:
Great suggestion - will fix in O(1h)

Reply:
Fixed.